File: app/src/main/java/maw/org/wedding/map/MarkerViewController.java
package maw.org.wedding.map;

import com.google.android.gms.maps.model.Marker;

public interface MarkerViewController {
    void updateViewModelForMarker(Marker marker, MarkerViewModel markerViewModel);
    boolean markerHasData(String markerID);

    MarkerViewModel getMarkerViewModel(String id);
}

File: app/src/main/kotlin/maw/org/wedding/map/MarkerViewController.kt
package maw.org.wedding.map

import com.google.android.gms.maps.model.Marker

interface MarkerViewController {
    fun updateViewModelForMarker(marker: Marker, markerViewModel: MarkerViewModel)
    fun markerHasData(markerID: String): Boolean

    fun getMarkerViewModel(id: String): MarkerViewModel
}
